test.sh

```diff
@@ -19,6 +19,7 @@ mkdir -p test
 for name in "blog-post" "blog-post2"; do
     echo "Test $name"
     go run main.go \
+       -c example/default.config.yaml \
```

Comment:
Comment: The config file option can be removed once the defaulting logic is implemented.

Reply:
I has issues running the test with the default config causing the test and example to differ.
Later realized I still had `category` at `json:"category,omitempty"` here https://github.com/TheGroundZero/tcardgen/commit/5800ea70c34d29c37522f59edf5c42b2962bface#diff-a3d824da3c42420cd5cbb0a4a2c0e7b5bfddd819652788a0596d195dc6e31fa5R14

Also, the default config file differs from the default config. `bgHexColor` for tags is different.
https://github.com/TheGroundZero/tcardgen/commit/582e6117ba5ea3b2cee0f68492adc9a8b6568051#diff-2a1cfff6cfb4e9eb85bc679eadae376e87b6a775ab49da85456ceb6aaa7beb3cR35